src/components/BlacklistedTagsSelector/index.tsx

```diff
@@ -0,0 +1,408 @@
+  searchKeywords: 'Search keywords…',
+  starttyping: 'Starting typing to search.',
+  nooptions: 'No results.',
+  blacklistedTagImportTitle: 'Import Blacklisted Tag Configuration',
+  blacklistedTagImportInstructions: 'Paste blacklist tag configuration below.',
+  valueRequired: 'You must provide a value.',
+  noSpecialCharacters:
+    'Configuration must be a comma delimited list of TMDB keyword ids, and must not start or end with a comma.',
+  invalidKeyword: '{keywordId} is not a TMDB keyword.',
+});
+
+type SingleVal = {
+  label: string;
+  value: number;
+};
+
+type BlacklistedTagsSelectorProps = {
+  defaultValue?: string;
+};
+
+const BlacklistedTagsSelector = ({
+  defaultValue,
+}: BlacklistedTagsSelectorProps) => {
+  const { setFieldValue } = useFormikContext();
+  const [value, setValue] = useState<string | undefined>(defaultValue);
+  const intl = useIntl();
+  const [selectorValue, setSelectorValue] =
+    useState<MultiValue<SingleVal> | null>(null);
+
+  const update = useCallback(
+    (value: MultiValue<SingleVal> | null) => {
+      const strVal = value?.map((v) => v.value).join(',');
+      setSelectorValue(value);
+      setValue(strVal);
+      setFieldValue('blacklistedTags', strVal);
+    },
+    [setSelectorValue, setValue, setFieldValue]
+  );
+
+  const copyDisabled = value === null || value?.length === 0;
+
+  return (
+    <>
+      <ControlledKeywordSelector
+        value={selectorValue}
+        onChange={update}
+        defaultValue={defaultValue}
+        components={{
+          DropdownIndicator: undefined,
+          IndicatorSeparator: undefined,
+          ClearIndicator: VerifyClearIndicator,
+        }}
+      />
+
+      <CopyButton
+        textToCopy={value ?? ''}
+        disabled={copyDisabled}
+        toastMessage={intl.formatMessage(messages.copyBlacklistedTags)}
+        tooltipContent={intl.formatMessage(
+          copyDisabled
+            ? messages.copyBlacklistedTagsEmpty
+            : messages.copyBlacklistedTagsTip
+        )}
+        tooltipConfig={{ followCursor: false }}
+      />
+      <BlacklistedTagsImportButton setSelector={update} />
+    </>
+  );
+};
+
+type BaseSelectorMultiProps = {
+  defaultValue?: string;
+  value: MultiValue<SingleVal> | null;
+  onChange: (value: MultiValue<SingleVal> | null) => void;
+  components?: Partial<typeof components>;
+};
+
+const ControlledKeywordSelector = ({
+  defaultValue,
+  onChange,
+  components,
+  value,
+}: BaseSelectorMultiProps) => {
+  const intl = useIntl();
+
+  useEffect(() => {
+    const loadDefaultKeywords = async (): Promise<void> => {
+      if (!defaultValue) {
+        return;
+      }
+
+      const keywords = await Promise.all(
+        defaultValue.split(',').map(async (keywordId) => {
+          try {
+            const { data } = await axios.get(`/api/v1/keyword/${keywordId}`);
+            return data;
+          } catch (err) {
+            throw new Error('Network repsonse was not ok');
+          }
```

Comment:
You can remove the try/catch since it's just catching the error from axios to throw another less explicit error